packages/shared/src/hooks/useValueRef.ts

```diff
@@ -12,3 +13,19 @@ export function useValueRef<T>(ref: ValueRef<T>) {
 
     return useSubscription(subscription)
 }
+
+export function useValueRefDelayed<T>(ref: ValueRef<T>, latency = 500) {
```

Comment:
Can you describe what is this used for? I know the functionality of this but I don't know the context of using this.